runtime/tools/spec-builder/test/utils/spawn-child-process.js

```diff
@@ -0,0 +1,47 @@
+const { spawn } = require('child_process');
+
+class Utils{
+    static buildSpec(file, output) {
+        return new Promise(function(resolve, reject) {
+            const defaults = {
+                cwd: process.cwd(),
+                env: process.env
+            }
+            const command = spawn('node', ['index.js', '-f', file, '-o', output], defaults);
+        
+            command.stdout.on('data', (data) => {
+                resolve(data);
+            })
+        
+            command.stderr.on('error', (error) => {
+                reject(error);
+            })
+            command.on('close', function(code){
+                resolve(code);
+            })
+        })
+    }
+
+    static emptyActualRawFiles() {
+        return new Promise(function(resolve, reject) {
+
+            const args = ['-rf', 
```

Comment:
why don't we just delete the `test/actual-raw-files` folder?

Reply:
because buildspec will fail to create a new file in non-existing directory

Reply:
I think that you can easily mitigate that problem using something like:
```JavaScript
if (!fs.existsSync(dir)){
    fs.mkdirSync(dir);
}
```